lib/ProductOpener/Display.pm

```diff
@@ -817,7 +821,13 @@ sub analyze_request($)
 	elsif ($components[0] eq "search") {
 		$request_ref->{search} = 1;
 	}
-	
+
+	# /tags API endpoint
```

Comment:
Is it "tags", or "taxonomy"?

Reply:
Good catch, it's taxonomy, I changed my mind half way through coding. I'll fix it.